cs-config.php

```diff
@@ -30,7 +30,7 @@
     'phpdoc_single_line_var_spacing' => false,
     'phpdoc_summary' => false,
     'phpdoc_var_annotation_correct_order' => false,
-    'php_unit_internal_class' => [],
+    'php_unit_internal_class' => ['types' => ['normal', 'final', 'abstract']],
```

Comment:
Have you tried to remove this option btw 🤔 
What this setting is going to do, is actually going to add @internal docblock on all test classes 🙈